controllers/operands/kubevirtConsolePlugin.go

```diff
@@ -195,12 +204,37 @@ func getKvUIDeployment(hc *hcov1beta1.HyperConverged, deploymentName string, ima
 		}
 	} else {
 		deployment.Spec.Template.Spec.NodeSelector = nil
-		deployment.Spec.Template.Spec.Affinity = nil
+		deployment.Spec.Template.Spec.Affinity = affinity
 		deployment.Spec.Template.Spec.Tolerations = nil
 	}
 	return deployment
 }
 
+func getPodAntiAffinity(componentLabel string, infrastructureHighlyAvailable bool) *corev1.Affinity {
+	if infrastructureHighlyAvailable {
+		return &corev1.Affinity{
+			PodAntiAffinity: &corev1.PodAntiAffinity{
+				RequiredDuringSchedulingIgnoredDuringExecution: []corev1.PodAffinityTerm{
+					{
+						LabelSelector: &metav1.LabelSelector{
+							MatchExpressions: []metav1.LabelSelectorRequirement{
+								{
+									Key:      hcoutil.AppLabelComponent,
+									Operator: metav1.LabelSelectorOpIn,
+									Values:   []string{componentLabel},
+								},
+							},
+						},
+						TopologyKey: corev1.LabelHostname,
+					},
+				},
+			},
+		}
+	}
+
+	return nil
+}
```

Comment:
not sure I want this function here. we only use it in the new tests. let's move it there

Reply:
where should i move it to? `testUtils.go` ?

Reply:
to the only file that uses it

Reply:
could you please suggest where this function should be move into?
note that it isn't used by tests only.

Reply:
>to the only file that uses it

the only usage of that function is in the same file - `kubevirtConsolePlugin.go`

Reply:
sorry. misread